TACC/Frontera-Portal
# Task/FP-768 move onboarding route

## Overview: ##

Move onboarding route under workbench

## PR Status: ##


## Related Jira tickets: ##

* [FP-768](https://jira.tacc.utexas.edu/browse/FP-768)

## Summary of Changes: ##

1. Move onboarding route under workbench
2. Remove middleware check of setup_complete
3. Provide initial setup_complete information in django template so we know immediately what it is.


## Testing Steps: ##
1. Follow steps of other onboarding  PR
2. Check that front end routes are unaccessible if your onboarding isn't completed.
3. After onboarding is complete, check that you can access all of workbench as expected.

## UI Photos:

![Screen Shot 2020-11-03 at 11 08 34 AM](https://user-images.githubusercontent.com/8287580/98017480-f10eb300-1dc4-11eb-94e5-a39698045723.png)

## Notes: ##
TODO
- [x] determine which backend request will have setup_complete. (api/workbench?)
- [x] Create task providing route/link for staff to access admin onboarding. _**[Going to create a ticket; discussing with Joon Yee]**_
